src/common/integration_test_base_class.py

```diff
@@ -0,0 +1,79 @@
+# standard library
+import unittest
+
+# third party
+import mysql.connector
+
+# first party
+from delphi.epidata.client.delphi_epidata import Epidata
+from delphi.epidata.server._limiter import limiter
+
+
+class BasicIntegrationTest(unittest.TestCase):
+    """Basic integration test class"""
+
+    def __init__(self, methodName: str = "runTest") -> None:
+        super().__init__(methodName)
+        self.delete_from_tables_list = []
+        self.truncate_tables_list = []
+        self.create_tables_list = []
+        self.role_name = None
+        self.epidata = Epidata
+        self.epidata.BASE_URL = "http://delphi_web_epidata/epidata/api.php"
+        self.epidata.auth = ("epidata", "key")
+
+    def delete_from_table(self, cur, table_name: str) -> None:
+        cur.execute(f"DELETE FROM `{table_name}`")
+
+    def truncate_table(self, cur, table_name: str) -> None:
+        cur.execute(f"TRUNCATE TABLE `{table_name}`")
+
+    def create_table(self, cur, create_table_stmt: str):
+        cur.execute(create_table_stmt)
+
+    def create_key_with_role(self, cur, role_name: str):
+        cur.execute("TRUNCATE TABLE `user_role`")
+        cur.execute("TRUNCATE TABLE `user_role_link`")
```

Comment:
you should move these two `TRUNCATE`s to underneath `DELETE FROM api_user` in `setUp()`, so the table truncation only happens once per test, and then we should also be able to add multiple roles if needed in the future.